types/call-bind/ts3.9/test/callBind.test.ts

```diff
@@ -13,7 +13,3 @@ applyBind(() => {}); // $ExpectType (thisArg: unknown, args: readonly []) => voi
 applyBind(Object.prototype.hasOwnProperty);
 // $ExpectType (args: readonly [string | number | symbol]) => boolean || (args: readonly [v: string | number | symbol]) => boolean
 applyBind(Object.prototype.hasOwnProperty, any);
-
-// These are invalid because of `package.json#exports`:
-import callBindIllegal1 = require('call-bind/index'); // $ExpectError
-import callBindIllegal2 = require('call-bind/index.js'); // $ExpectError
```

Comment:
@orta @sandersn @weswigham

Why do these lines cause `@types/call‑bind/index.d.ts` to be included in the **TypeScript 3.9** test?